docs/advanced/index.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: index
+title: Advanced
+navigation_weight: 1
+---
+
+# Caching CDN Requests or Non-200 Responses
+
+If an attempt is made to cache an asset that is on a different origin or
+the asset returns a non-200 response, these modules will throw an error. The
+reason for this default behavior is to ensure that only good and local
+responses are cached.
+
+Depending on how you are using these libraries will depend on how you can add
+support for external resources.
+
+## sw-lib
+
+In sw-lib, you can set up some additional options with a route that will allow
+support for additional status codes and / or based on specific headers.
+
+```javascript
+const cdnCacheStrategy = goog.swlib.staleWhileRevalidate({
+  cacheableResponse: {
+    statuses: [0],
+  },
+});
+goog.swlib.router.registerRoute(new RegExp('^https://cdn.mysite.com/styles/'), cdnCacheStrategy);
+```
+
+This same approach can be used with all of th caching strategies supported by
```

Comment:
Fix 'th' > 'the'.

While you're editing, please use active voice. In other words, "use this approach" instead of "this approach can be used."